src/DIRAC/TransformationSystem/DB/TransformationDB.py

```diff
@@ -603,6 +603,7 @@ def getTransformationFiles(
         offset=None,
         connection=False,
         columns=None,
+        include_web_records=True,
```

Comment:
I would reverse this flag, and make a tiny WebApp patch.

Reply:
We suspect it isn't used at all. The idea would be to switch it back in v9 but it's a nasty backwards incompatible change in a patch release if anyone does happen to be using it.